docs/modules/jenkins-shared-library/partials/odsComponentStageRolloutOpenShiftDeployment.adoc

```diff
@@ -15,7 +15,7 @@ Available options:
 | Name of `DeploymentConfig` to use (defaults to `context.componentId`).
 
 | deployTimeoutMinutes
-| Adjust timeout of rollout (defaults to 5 minutes).
+| Adjust timeout of rollout (defaults to 5 minutes). Needs to be kept in sync with the OpenShift rollout timeouts and summarized health-check timeouts.
```

Comment:
What exactly are you referring to with "OpenShift rollout timeouts"?

Reply:
The timeoutSeconds parameter of the rolling deployment strategy

Reply:
Ah! Good point. Maybe we can say sth. like this then:

> Caution: needs to be aligned with the deployment strategy timeout (`timeoutSeconds`) and the readiness / liveness probe timeouts.

Reply:
Fine with me, will change it :)